### DISCUSSION TITLE
Don't receive the mail to acces on the admin pannel / Hubs by cloud

### DISCUSSION THREAD

**Discussion by MyVersep** [mozilla/hubs#5626] [Hubs Cloud Questions]
Hello

I don't success to receive a mail when I enter my admin address.

I create my stack, I type my domain name/admin, I enter my email address to access the admin part but I receive nothing

**Comment by matthewbcool** [comment]
Possibly the issue: Set up AWS Simple Email Service (SES) in **N. Virginia (us-east-1)**.

**Reply by MyVersep** [reply]
It is already set up

**Reply by pattersonbl2** [reply]
You can also look on the ses dashboard in aws to see email metrics.  This would be the starting place for troubleshooting after that I would look into the system logs of your server.

**Reply by matthewbcool** [reply]
It is already set up > 
@MyVersep 
confirming in N. Virginia (us-east-1)?

**Reply by pattersonbl2** [reply]
> 

By default,   SES should always deploy in the us-east-1 and I don't think users can change that at the moment.

**Comment by anuj-upadhyay-2001** [comment]
Facing the same issue, unable to receive the admin confirmation email. I have also setup Amazon SES (us-east-1) for both my domain and the admin email address.

**Reply by pattersonbl2** [reply]
You would often have to ssh into your server to view logs on this issue.  It's hard to tell from the outside what this issue could be. The system logs in your ec2 give a better view.
